internal/app/machined/pkg/controllers/kubespan/nftables.go

```diff
@@ -324,6 +370,119 @@ func matchIPSet(set *nftables.Set, mark, mask uint32, family nftables.TableFamil
 	}
 }
 
+func matchIPSetMSS(set *nftables.Set, mtu uint32, family nftables.TableFamily) []expr.Any {
+	offset, length := ipOffsetLength(family)
+
+	var mss uint16
+
+	switch family { //nolint:exhaustive
+	case nftables.TableFamilyIPv4:
+		mss = uint16(mtu) - 40 // TCP + IPv4 overhead
```

Comment:
maybe a comment on what 40 and 60 means would be helpful

Reply:
isn't this what I added on this line?

Reply:
missed it :sweat_smile: